src/interfaces/pubsub.js

```diff
@@ -121,17 +121,10 @@ class PubSub {
                 // If replay event: publish all the data matched from event queue
                 let publishData = message;
 
-                if (subscription.type === "RE_PLAY") {
-                    publishData = eventQ.store.filter((evt)=> {
-                        if (evt.publisher === publisher && evt.eventName === eventName) {
-                            return evt;
-                        }
-                    }).map((evt)=> {
-                        return evt.message;
-                    });
-                }
+				if( (context.lifeCycleFlags &&  context.lifeCycleFlags.rendered == true ) || ( context.initOn && context.initOn.eventName == eventName ) ){
```

Comment:
Because of this condition, I guess KEEP_ON events wont be triggered. KEEP_ON events can should get triggered even if Module is not rendered.